keras_cv/layers/preprocessing/cut_mix.py

```diff
@@ -59,9 +60,9 @@ def _batch_augment(self, inputs):
                 f"Got: inputs = {inputs}"
             )
         images, labels = self._update_labels(*self._cutmix(images, labels))
-        inputs["images"] = images
-        inputs["labels"] = labels
-        return inputs
+        #inputs["images"] = images
```

Comment:
Let's revert these changes and keep the PR localized to the docstring update.  Thanks!

Reply:
> 
Reverted changes.